man/Makefile

```diff
@@ -0,0 +1,11 @@
+PAGE_SRCS := $(shell find -name '*.[0-9].txt' -printf '%P\n')
+PAGES := $(patsubst %.txt,output/%,$(PAGE_SRCS))
+
+all: $(PAGES)
+
+output/%: %.txt
+	@mkdir -p $$(dirname '$@')
+	@echo [GEN] $@ ; manscript -o $@ $<
```

Comment:
Where does this `manscript` come from? Never heard about it.

Reply:
https://github.com/gphalkes/tilde/tree/master/man

Reply:
Maybe we should use old school turbo.1.in https://github.com/amay5267/turbo/blob/amay5267-patch-1/man/turbo.1.in

Reply:
Can you please write proper cross-platform Makefile for this MAN?
I can test and review that on macOS as well.